---
layout: pages
title: Publications
permalink: /publications/
image: /img/heterogeneity.png
---

### Publications
___

#### Preprints

<div class="col-lg-11">
    <p>
    Banerjee A, Pattinson DJ, Wincek CL, Bunk P, Chapin SR, Navlakha S*, Meyer HV*
    (2024) <a href="https://www.biorxiv.org/content/10.1101/2024.01.22.576714v2">
    BATMAN: Improved T cell receptor cross-reactivity prediction
    benchmarked on a comprehensive mutational scan database</a>,
    bioRxiv, 2024.01.22.576714
    </p>
    <p>
    Kovaleva VA, Pattinson DJ, Barton C, Chapin SR, Minervina AA, Richards KA, Sant
    AJ, Thomas PG*, Pogorelyy MV*, Meyer HV* (2023) <a
    href="https://www.biorxiv.org/content/10.1101/2023.11.28.569052v2">copepodTCR:
    Identification of Antigen-Specific T Cell Receptors with combinatorial peptide
    pooling</a>, bioRxiv, 2023.11.28.569052
    </p>
    <p>
    Yamrom B, Lee Y, Marks S, Chorbadjiev L, Meyer HV, Iossifov I (2022) <a
    href="https://www.biorxiv.org/content/10.1101/2022.12.08.519599v1">Snakeobjects:
    an object-oriented workflow management system</a>, bioRxiv, 2022.12.08.519599
    </p>
</div>

#### Publications

<div class="col-lg-11">
    <p>
    Loh L, Carcy S, Krovi HS, Domenico J, Spengler A, Lin Y, Torres J, Prabakar R,
    Palmer W, Norman PJ, Stone M, Brunetti T, Meyer HV*, Gapin L (2024) Unraveling
    the Phenotypic States of Human innate-like T Cells:
    <a href="https://www.sciencedirect.com/science/article/pii/S2211124724010568?via%3Dihub">
    Comparative Insights with Conventional T Cells and Mouse Models</a>, Cell Reports
    </p>
    <p>
    Kleeman SO, Ferrer M, Demestichas B, Bankier S, Lee H, Heywood T, Ruusalepp A,
    Bjorkegren JLM, Walker BR,
    Kleeman SO, Thakir TM, Demestichas B, Mourikis N, Loiero D, Ferrer M, Bankier S,
    Riazat-Kesh YJRA, Lee H, Chantzichristos D, Regan C, Preall J, Sinha S, Rosin N,
    Yipp B, de Almeida LGN, Biernaskie J, Dufour A, Tober-Lau P, Ruusalepp A,
    Bjorkegren JLM, Ralser M, Kurth F, Demichev V, Heywood T, Gao Q, Johannsson G,
    Koelzer VH, Walker BR, Meyer HV*, Janowitz T* (2023) <a
    href="https://www.sciencedirect.com/science/article/pii/S2666979X23001222?via%3Dihub">
    Cystatin C is glucocorticoid responsive, directs recruitment of Trem2+ macrophages,
    and predicts failure of cancer immunotherapy</a>,
    Cell Genomics
    </p>
    <p>
    Mou H, Eskiocak O, Özler KA, Gorman M, Yue J, Jin Y, Wang Z, Gao Y, Janowitz T,
    Meyer HV, Yu T, Wilkinson JE, Kucukural A, Ozata DM, Beyaz S. (2023) <a
    href="https://pathsocjournals.onlinelibrary.wiley.com/doi/full/10.1002/path.6054">
    CRISPR-induced exon skipping of β-catenin reveals tumorigenic mutants driving
    distinct subtypes of liver cancer</a>. J Pathol (4):415-427
    </p>
    <p>
    Carter JA, Strömich L, Peacey M, Chapin SR, Velten L, Steinmetz LM, Brors B,
    Pinto S & Meyer HV (2022)
    <a href="https://www.nature.com/articles/s41467-022-31750-1">
    Transcriptomic diversity in human medullary thymic epithelial cells.
    </a> Nature Communications
    </p>
    <p>
    Kleeman SO, Cordioli M, Timmers PRHJ, Khan A, Tober-Lau P, Kurth F, Demichev V,
    Meyer HV, Wilson JF, Ralser M, Kiryluk K, Ganna A, Baillie K, Janowitz T (2022)
    <a href="https://www.sciencedirect.com/science/article/pii/S2589004222013128">
    Cystatin C is associated with adverse COVID-19 outcomes in diverse
    populations</a>, IScience, 25(10):105040
    </p>
    <p>
    Beyaz S, Roper J, Bauer-Rowe KE, Xifaras ME, Ergin I, Dohnalova L, Biton M,
    Shekar K, Mou H, Eskiocak O, Özata DM, Papciak K, Chung C, Almeqdadi M, Fein M,
    Valle-Encinas E, Erdemir A, Dogum K, Alici-Garipcan A, Meyer HV, Fox JG, Elinav E,
    Kucukural A, Kumar P, McAleer J, Thaiss CA, Regev A, Orkin SH, Yilmaz ÖH (2020)
    <a href="https://www.sciencedirect.com/science/article/pii/S1934590921003441?via%3Dihub">
    Dietary suppression of MHC-II expression in intestinal stem cells enhances
    intestinal tumorigenesis</a>, 2021, Cell Stem Cell
    </p>
    <p>
    Williams EH, Flint, TH’ Connell CM, Giglio D, Lee H, Ha T, Gablenz E, Bird N,
    Weaver JMJ, Potts H, Whitley CT, Bookman MA,  Lynch AG, Meyer HV, Tavaré
    S, Janowitz T (2020) <a
    href="https://clincancerres.aacrjournals.org/content/early/2020/12/08/1078-0432.CCR-20-3201">CamGFR
    v2: a new model for estimating the glomerular filtration rate from standardized
    or non standardized creatinine in patients with cancer</a>, Clinical Cancer Research
    </p>
    <p>
    Meyer HV*, Dawes TJ, Serrani M, Wenjia B, Tokarczuk P, Jiashen C, de Marvao A, Henry A,
    Lumbers RT, Gierten J, Thumberger T, Wittbrodt J, Ware JS, Rueckert D, Matthews PM, Prasad SK,
    Costantino ML, Birney E, Cook SA and O'Regan DP* (2020) <a
    href="https://www.nature.com/articles/s41586-020-2635-8">Genetic and functional insights into the fractal
    structure of the heart.</a> Nature, 584:589- 594
    </p>

    <p>
    Meyer HV, Casale FP, Stegle O, Birney E (2018)
    <a href="https://www.biorxiv.org/content/early/2018/01/30/255497">LiMMBo:
    a simple, scalable approach for linear mixed models in high-dimensional genetic
    association studies</a>, <i>bioRxiv</I>
    </p>

    <p>
    Meyer HV, Birney E (2018) <a
    href="https://doi.org/10.1093/bioinformatics/bty197">PhenotypeSimulator: a
    comprehensive framework for multi-trait, multi-locus phenotype to genotype
    simulation</a>, <i>Bioinformatics</i>, bty197
    </p>

    <p>
    Biffi C, de Marvao A,  Attard MI, Dawes TJW, Whiffin N,  Bai W, Shi W,
    Francis C, Meyer HV, Buchan R, Cook SA, Rueckert D, O’Regan DP (2017)
    <a href="https://academic.oup.com/bioinformatics/article/doi/10.1093/bioinformatics/btx552/4103396/Threedimensional-cardiovascular-imaginggenetics-a">Three-dimensional cardiovascular imaging-genetics: a mass univariate framework</a>, <i>Bioinformatics</i>, btx552
    </p>


    <p>
    Keil M, Sonner JK, Lanz TV, Oezen I, Bunse T, Bittner S, Meyer HV, Meuth SG,
    Wick W, Platten M. (2016)
    <a href="http://www.jni-journal.com/article/S0165-5728(16)30119-9/abstract">General control non-derepressible 2 (GCN2) in T cells controls disease progression of autoimmune neuroinflammation</a>, <i>Journal of Neuroimmunology</i>, 297:117-126
    </p>


    <p>
    Rattay K, Meyer HV, Herrmann C, Brors B, Kyewski B. (2015) <a href="http://www.sciencedirect.com/science/article/pii/S0896841115300469?via\%3Dihub0">Evolutionary conserved gene co-expression drives generation of self-antigen diversity in medullary thymic epithelial cells</a>, <i>Journal of Autoimmunity</i>, 67:65-75
    </p>

    <p>
    Beer R, Herbst K, Ignatiadis N, Kats I, Adlung L, Meyer HV, Niopek D, Christiansen T, Georgi F, Kurzawa N, Meichsner J, Rabe S, Riedel A, Sachs J, Schessner J, Schmidt F, Walch P, Niopek K, Heinemann T, Eils R, Di Ventura B. (2014) <a href="http://pubs.rsc.org/en/Content/ArticleLanding/2014/MB/C3MB70594C#!divAbstract">Creating functional engineered variants of the single-module non-ribosomal peptide synthetase IndC by T domain exchange</a>, <i>Molecular BioSystems</i>, 10:1709-1718
    </p>

    <p>
    Renner M, Wolf T, Meyer HV, Hartmann W, Penzel R, Ulrich A, Lehner B, Hovestadt V, Czwan E, Egerer G, Schmitt T, Alldinger I, Renker EK, Ehemann V, Eils R, Wardelmann E, Büttner R, Lichter P, Brors B, Schirmacher P, Mechtersheimer G. (2013) <a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2013-14-12-r137">Integrative DNA methylation and gene expression analysis in high-grade soft tissue sarcomas.</a>
    <i>Genome Biology</i>, 14:r137
    </p>
</div>
